### ISSUE TITLE
It is invalid to reorder `RTCRtpEncodingParameters` in `RTCRtpTransceiverInit`

### ISSUE THREAD

**Issue by mstyura** [flutter-webrtc/flutter-webrtc#646]
**Describe the bug**
The reordering of `RTCRtpEncodingParameters` when `RTCRtpTransceiverInit` is wrong thing to do.
The reordering of `RTCRtpEncodingParameters` is done in plugins for [`iOS`](https://github.com/flutter-webrtc/flutter-webrtc/blob/6b42c510e78dc767a50821ae995005129b748570/common/darwin/Classes/FlutterWebRTCPlugin.m#L1506-L1512) and [`Android`](https://github.com/flutter-webrtc/flutter-webrtc/blob/6b42c510e78dc767a50821ae995005129b748570/android/src/main/java/com/cloudwebrtc/webrtc/PeerConnectionObserver.java#L678-L681), but not for [`Web`](https://github.com/flutter-webrtc/flutter-webrtc/blob/6ae3c5b84c5f2555063d5bac73da803867b1b2de/lib/src/web/rtc_rtp_transceiver_impl.dart#L16-L18) and in `Web` browsers like [`Chromium` does not revert send encodings](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/peerconnection/rtc_rtp_transceiver.cc;l=90-110;drc=4142c1f8dd8219d2d5743c8126f60349a4ed101c;bpv=0;bpt=1) list before passing to native `WebRTC` as well.

It is also discussed [here](https://github.com/w3c/webrtc-pc/issues/1896) and from the discussion it can be concluded that the ordering of send encodings affects the behavior of WebRTC and hence users should be able to control it, so plugin must not do any change to encodings order. 

**To Reproduce**
Use `RTCPeerConnection.addTransceiver(.., RTCRtpTransceiverInit)` overload and specify list of send encodings `RTCRtpEncodingParameters`. The order of encodings passed to native `WebRTC` in reverse order. The change was introduced by this commit https://github.com/flutter-webrtc/flutter-webrtc/commit/40bde2ae3db08691427cb474db48618754256083.

**Expected behavior**
The order of `RTCRtpEncodingParameters` is preserved and passed as is to native `WebRTC`.

**Platform information**

* **Flutter version**: 
* **Plugin version**: `0.6.5`
* **OS**: Android & iOS 
* **OS version**:

**Comment by cloudwebrtc** [comment]
Yes, I noticed this problem very early. When implementing the simulcast function, I must sort, otherwise, the simulcast stream cannot be created, you try to restore https://github.com/flutter-webrtc/flutter-webrtc/commit /40bde2ae3db08691427cb474db48618754256083 in the modification, and test, encoding sorting, simulcast, etc.
